todolist/settings.py

```diff
@@ -66,7 +66,7 @@
         'NAME': 'app_db',
         'USER': 'app_user',
         'PASSWORD': '1234',
```

Comment:
Hardcoding database credentials is insecure. Use environment variables or a configuration management tool to manage sensitive information securely.